_content/monographs/satipatthana-meditation-practice-guide_analayo.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Satipaṭṭhāna Meditation: A Practice Guide"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/satipatthanapracticeguide.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1pBhjHfVWRqLxMsneSjSt2FLFNU_SdLkk/view?usp=sharing"
+course: sati
+tags:
```

Comment:
I see, you just tagged each of the chapters... hmmm... instead of doing that, let's tag with `meditation`* and `vipassana` Maybe we can keep `feelings` as well...

\* - I know normally we don't tag a parent and child, but I think I'll make an exception here to raise its profile a bit...